src/Mod/Draft/Draft.py

```diff
@@ -387,7 +387,8 @@ def makeWire(pointslist,closed=False,placement=None,face=None,support=None):
     if placement:
         typecheck([(placement,FreeCAD.Placement)], "makeWire")
         ipl = placement.inverse()
-        pointslist = [ipl.multVec(p) for p in pointslist]
+        if bs2wire is not True:
```

Comment:
It's more readable if you just say 
```
if bs2wire is False:
```
or more Pythonic
```
if not bs2wire:
```

Reply:
Changed to more Pythonic suggestion